src/main/java/teammates/ui/webapi/CreateFeedbackSessionLogAction.java

```diff
@@ -17,7 +25,31 @@ void checkSpecificAccessControl() {
 
     @Override
     JsonResult execute() {
-        // TODO: implement the logic for creating a feedback session log
+        String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
+        String fsName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
+        FeedbackSessionAttributes feedbackSession = getNonNullFeedbackSession(fsName, courseId);
+        if (feedbackSession == null) {
+            return new JsonResult("No feedback session with name " + fsName
+                    + " for course " + courseId, HttpStatus.SC_NOT_FOUND);
+        }
+
+        String fslType = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_LOG_TYPE);
+        if (!fslType.equals(FeedbackSessionLogTypes.ACCESS) && !fslType.equals(FeedbackSessionLogTypes.SUBMISSION)) {
+            return new JsonResult("Invalid feedback session log type.", HttpStatus.SC_BAD_REQUEST);
+        }
+
+        String studentEmail = getNonNullRequestParamValue(Const.ParamsNames.STUDENT_EMAIL);
```

Comment:
what if there is no email but only a registration key?

Reply:
According to https://github.com/TEAMMATES/teammates/pull/10984#discussion_r589159208, there should be no issues with the frontend in retrieving email even if only reg key is used